.travis.yml

```diff
@@ -26,6 +26,14 @@ matrix:
     env: PUPPET_VERSION="~> 3.6.0"
   - rvm: 2.2.3
     env: PUPPET_VERSION="~> 3.7.0"
+  - rvm: 2.3.3
+    env: PUPPET_VERSION="~> 3.4.0"
+  - rvm: 2.3.3
+    env: PUPPET_VERSION="~> 3.5.0"
+  - rvm: 2.3.3
+    env: PUPPET_VERSION="~> 3.6.0"
+  - rvm: 2.3.3
+    env: PUPPET_VERSION="~> 3.7.0"
```

Comment:
I think we should remove these old versions of Puppet. https://github.com/dfarrell07/puppet-opendaylight/commit/58fd3bfdd4be4f3c24679d57ca7bb7a4e4f2d833#diff-354f30a63fb0907d4ad57269548329e3R18